MISampleApp/SFMISampleConsole/run.ps1

```diff
@@ -0,0 +1,11 @@
+param([string]$mode="env:cert")
+
+# Docker image name for the application
```

Comment:
This file can be left out, it's for local testing purposes. (And won't run without the SF-supplied env vars.)